backend: move the win_bind_shadow function call

absolutelynothelix opened the pull request:
the win_bind_shadow function must be called before checking if a window has shadow enabled because it disables shadow for a window on failure.

absolutelynothelix commented:
i forgor to rename the branch :skull:

yshui commented on src/backend/backend.c:
```diff
@@ -332,16 +332,16 @@ bool paint_all_new(session_t *ps, struct managed_win *const t) {
 		}
 
+		if (w->shadow && !w->shadow_image) {
+			struct color shadow_color = {.red = ps->o.shadow_red,
```
can you add a comment here?

absolutelynothelix replied:
done

absolutelynothelix committed:
backend: move the win_bind_shadow function call

the win_bind_shadow function must be called before checking if a window
has shadow enabled because it disables shadow for a window on failure.

yshui reviewed (approved):
Nice
